web-app/mock-data/request-body.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Fave",
```

Comment:
@rashmi-carol-dsouza @aaron-to-go 

1. how will the name be used by you?
2. Is it possible to find Fave by name or do you actually need an artist ID for reliable results?

Reply:
By name you mean artist name, right?
1. I don't think that we'll use the name for the search, since we don't need it for the chartmetric endpoint getArtistListByFilters
2. No, usually all queries require some chartmetric id

Reply:
Ah that clears it out then. Thank you!
For now I will just display the name to offer a feel of personalised results :)